lib/models/application.js

```diff
@@ -1564,6 +1567,12 @@ const getApplicationModel = function (deps, opts) {
 			 */
 			remove: envVarModel.remove,
 		}),
+
+		/**
+		 * @namespace balena.models.application.invite
+		 * @memberof balena.models.application
+		 */
+		invite: addCallbackSupportToModule(inviteModel),
```

Comment:
Are you sure that this doesn't need to be 
```suggestion
		invite: addCallbackSupportToModule(inviteModel()),
```

?? We will not have to do this change though if we drop the `once`.